content/asset-tracking/using-the-sdks.textile

```diff
@@ -72,15 +72,21 @@ The client requires authentication in order to establish a connection with Ably.
 
 Usually a client will use either token or JWT authentication, as basic authentication would require exposing the API keys on the client.
 
-Examples of establishing a connection using the three methods are given in the following sections. While the examples shown are for either the Publishing or Suscribing SDK, you can use the same approach for both SDKs.
+Examples of establishing a connection using the three methods are given in the following sections. While the examples shown are for either the Publishing or Subscribing SDK, you can use the same approach for both SDKs.
 
 h3(#basic-authentication). Basic Authentication
 
 The following example demonstrates establishing a connection using basic authentication:
 
 ```[android]
 val publisher = Publisher.publishers() // get the Publisher builder in default state
-    .connection(ConnectionConfiguration(Authentication.basic(CLIENT_ID, ABLY_API_KEY)))
+  .connection(ConnectionConfiguration(Authentication.basic(CLIENT_ID, ABLY_API_KEY)))
+```
+
+```[swift]
+publisher = try? PublisherFactory.publishers() // get a Publisher builder
```

Comment:
It might be nice to replace `publisher` with `let publisher`. Then all the question marks (e.g. `try?` here, and also `
publisher?.functionName` can be replaced with `publisher.functionName`)

Reply:
Good idea. Changed.